packages/amplication-design-system/package.json

```diff
@@ -31,22 +31,24 @@
   },
   "devDependencies": {
     "@babel/core": "7.18.10",
-    "@storybook/addon-actions": "6.4.22",
-    "@storybook/addon-essentials": "6.4.22",
-    "@storybook/addon-interactions": "6.4.22",
-    "@storybook/addon-links": "6.5.10",
-    "@storybook/builder-webpack5": "6.5.9",
-    "@storybook/manager-webpack5": "6.4.22",
-    "@storybook/preset-scss": "1.0.3",
-    "@storybook/react": "6.5.10",
-    "@storybook/testing-library": "0.0.13",
+    "@storybook/addon-actions": "^6.5.10",
+    "@storybook/addon-essentials": "^6.5.10",
+    "@storybook/addon-interactions": "^6.5.10",
+    "@storybook/addon-links": "^6.5.10",
+    "@storybook/builder-webpack4": "^6.5.10",
+    "@storybook/builder-webpack5": "^6.5.10",
+    "@storybook/manager-webpack4": "^6.5.10",
+    "@storybook/manager-webpack5": "^6.5.10",
+    "@storybook/preset-scss": "^1.0.3",
+    "@storybook/react": "^6.5.10",
+    "@storybook/testing-library": "^0.0.13",
     "@types/classnames": "2.3.1",
     "@types/lodash": "4.14.178",
     "@types/react": "17.0.34",
     "@typescript-eslint/eslint-plugin": "5.9.1",
     "@typescript-eslint/parser": "5.9.1",
     "babel-loader": "8.2.5",
-    "css-loader": "6.7.1",
+    "css-loader": "^6.7.1",
```

Comment:
Why is this not anchor?

Reply:
Locked versions don't let to update packages that dependent packages. Basically that what didn't let storybook to compile and run.